packages/core/src/components/forms/textArea.tsx

```diff
@@ -73,11 +83,14 @@ export class TextArea extends AbstractPureComponent<TextAreaProps, TextAreaState
     );
 
     private maybeSyncHeightToScrollHeight = () => {
-        if (this.props.growVertically && this.textareaElement != null) {
+        if (this.props.autoResize && this.textareaElement != null) {
```

Comment:
we need to continue supporting the `growVertically` prop when it's deprecated. otherwise that's not a deprecation, it's a breaking change.

Reply:
fixed